app/models/manageiq/providers/ansible_tower/shared/automation_manager/configuration_script_source.rb

```diff
@@ -60,7 +60,7 @@ def update_in_provider(params)
   end
 
   def update_in_provider_queue(params)
-    action = "Updating #{self.class.name} with manager_ref=#{manager_ref}"
+    action = "Updating #{self.class.name} with Tower internal reference=#{manager_ref}"
```

Comment:
I think `manager_ref` wouldn't mean much to an end user as seen [here](https://bugzilla.redhat.com/attachment.cgi?id=1268946).